docs/src/turing_simple.md

```diff
@@ -4,21 +4,23 @@ It is possible to use [Turing.jl](https://turinglang.org/stable/) to perform Bay
 
 Note that you can easily swap the LBA model from this example for other [SSM models](https://itsdfish.github.io/SequentialSamplingModels.jl/dev/api/) simply by changing the names of the parameters.
 
+```@setup turing_mle
+using Random
 
+Random.seed!(45461)
+```
 
 ## Load Packages
 
 The first step is to load the required packages. You will need to install each package in your local
 environment in order to run the code locally. We will also set a random number generator so that the results are reproducible.
 
-```@setup turing_simple
+```@example turing_simple
 using Turing
 using SequentialSamplingModels
 using Random
 using LinearAlgebra
 using StatsPlots
-
-Random.seed!(45461)
```

Comment:
I think the rng seed should be visible to readers so they can reproduce the results in the documentation.